python/seldon_core/microservice.py

```diff
@@ -169,6 +169,30 @@ def setup_tracing(interface_name: str) -> object:
     return config.initialize_tracer()
 
 
+def setup_logger(log_level: str) -> logging.Logger:
+    # set up log level
+    log_level_raw = os.environ.get(LOG_LEVEL_ENV, log_level.upper())
+    log_level_num = getattr(logging, log_level_raw, None)
+    if not isinstance(log_level_num, int):
+        raise ValueError("Invalid log level: %s", log_level)
+
+    logger.setLevel(log_level_num)
+
+    # Set right level on access logs
+    flask_logger = logging.getLogger("werkzeug")
+    flask_logger.setLevel(log_level_num)
```

Comment:
wonder if this will also solve https://github.com/SeldonIO/seldon-core/issues/1907 ?

Reply:
Tangentially. But not in the best way.

I flagged this earlier. That change means that no access logs will be shown (including the `/predict` requests). This may make sense on some scenarios, since those entries can also be logged in other places (i.e. tracing, executor, Istio sidecar, etc.). However, it is a breaking change since I assume some users will expect those logs to be there.

We should probably keep those (maybe make them optional under a different `--access-log` flag?).